src/chrome/handleWork.js

```diff
@@ -1,9 +1,10 @@
 import listenStoreChanges from '../listenStoreChanges';
 import { workFinished } from '../actions';
-import { mainAudioPlayer, playSound } from '../playSound';
+import { playSoundOptions, stopSound } from '../playSound';
 
 const handleWork = (store, chrome) => {
   let workTabId = null;
+  window.audioPlayer = null;
```

Comment:
I would make it module-local and would have to remove the test I wrote inside handleWork.js test.   Sounds sane?